vktrace/vktrace_common/vktrace_filelike.c

```diff
@@ -63,13 +63,34 @@ BOOL vktrace_Checkpoint_read(Checkpoint* pCheckpoint, FileLike* _in) {
 // ------------------------------------------------------------------------------------------------
 // ------------------------------------------------------------------------------------------------
 // ------------------------------------------------------------------------------------------------
+size_t vktrace_FileLike_GetFileLength(FILE* fp) {
```

Comment:
A side effect of this function is that it resets the file pos to the beginning of the file. Should add a comment warning about this.

Reply:
Maybe it is better to save file position at the beginning and reset it before return the file length in this function?
I didn't reset the file position because this function was only called by vktrace_FileLike_create_file() which means the file position is always the beginning of the file when calling this function.  I realized that it may confuse people now.  Will update it in next commit.

Reply:
Either way is fine - restore the file pos or add a comment that the file pos is changed when this function is called. I would lean toward the latter since the function is only called from one place.

Reply:
A comment warning about the reset of file position has been added.  Please help to review.